styles.css

```diff
@@ -0,0 +1,689 @@
+    flex-basis: 60px;
+    width: 100%;
+    flex-grow: 0;
+    flex-shrink: 0;
+    color: #FFF;
+    z-index: 1;
+}
+
+nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-basis: 400px;
+    list-style: none;
+}
+
+header .small-inner-container {
+    align-items: center;
+    flex-basis: 300px;
+    margin-top: 100px;
+    color: #FFF;
+}
+
+header button {
+    padding-top: 20px;
+    padding-bottom: 20px;
+    width: 280px;
+}
+
+header .squiggle-line-stripe {
+    position: absolute;
+    height: 120px;
+    bottom: 0;
+}
+
+/*Creates the tilted bottom on the header*/
+header::after {
+    content: "";
+    display: block;
+    width: 0;
+    height: 0;
+    border-top: 50px solid transparent;
+    border-left: 100vw solid #FFF;
+    position: absolute;
+    bottom: 0;
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+#introduction {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    position: relative;
+}
+
+#introduction .small-inner-container {
+    color: var(--purple);
+    text-align: center;
+    margin-bottom: 150px;
+    margin-top: 150px;
+}
+
+#introduction h2 {
+    margin-bottom: 30px;
+}
+
+#introduction p {
+    margin-top: 30px;
+}
+
+/*Adds floating words on the background*/
+#introduction::before {
+    content: "hello";
+    position: absolute;
+    font-weight: 800;
+    font-size: 175px;
+    color: var(--floating-words);
+    z-index: -1;
+    top: 80px;
+}
+
+/*Created tilted top at services, bottom of introduction*/
+#introduction::after {
+    content: "";
+    display: block;
+    width: 0;
+    height: 0;
+    border-bottom: 50px solid transparent;
+    border-right: 100vw solid #FFF;
+    position: absolute;
+    bottom: -49px;
+    z-index: 1;
+}
+
+/* --------------- 4.3 Studies ------------------- */
+#studies {
+    background-color: var(--project-background);
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    position: relative;
+    z-index: 0;
+}
+
+/*Adds floating words on the background*/
+#studies::before {
+    content: "work";
+    position: absolute;
+    font-weight: 800;
+    font-size: 175px;
+    color: #FFF;
+    z-index: -1;
+    top: 80px;
+}
+
+/*Creates the tilted bottom at services*/
+#studies::after {
+    content: "";
+    display: block;
+    width: 0;
+    height: 0;
+    border-top: 50px solid transparent;
+    border-left: 100vw solid #FFF;
+    position: absolute;
+    bottom: 0;
+}
+
+.bigger-inner-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    flex-basis: 100%;
+    margin: 220px 20px 50px;
+}
+
+#studies article {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 50px;
+    max-width: 500px;
+}
+
+#studies .article-content {
+    display: flex;
+    flex-direction: column;
+    order: 2;
```

Comment:
In plaats van werken met order, kun je ook gewoon de hele flex-direction van het omwikkelende article veranderen in `row-reverse` voor het 2e artikel. En dan bestaat er ook nog een CSS selector `nth-child(even)` en `nth-child(odd)`die je kan helpen om het voor alle 2e, 4e elementen toe te passen. Als je dan ooit nog een paar extra portfolio items toevoegen, blijft de CSS altijd werken